.github/workflows/reusable-delivery-issue-chores.yml

```diff
@@ -59,13 +59,22 @@ jobs:
       # Add all tickets of Type "Epic" and label "Area:" to the global roadmap project
       - name: Attach all Epics to the Roadmap - Global project
         uses: actions/add-to-project@v1.0.2
-        if: ${{ steps.get-issue.outputs.issue_type == 'Epic'}}
+        if: ${{ steps.get-issue.issue_type == 'Epic' }}}
```

Comment:
Are you sure we can refer to the return value of a previous step without using "outputs" ?

Reply:
my bad, I though there was a "_" between issue and outputs and than we made a mistake
thanks